TypeScript example fails to start

Looking at the main `README` file I was pointed towards: https://github.com/grpc/grpc-web/tree/master/net/grpc/gateway/examples/echo/ts-example

When running the specified command:

```
docker-compose up --build node-server envoy ts-client
```

The following happens:

```
[+] Running 5/5
 ✔ Network grpc-web-master_default          Created                                                          0.1s 
 ✔ Container grpc-web-master-prereqs-1      Created                                                          1.8s 
 ✔ Container grpc-web-master-node-server-1  Created                                                          0.1s 
 ✔ Container grpc-web-master-ts-client-1    Created                                                          0.2s 
 ✔ Container grpc-web-master-envoy-1        Created                                                          0.1s 
Attaching to grpc-web-master-envoy-1, grpc-web-master-node-server-1, grpc-web-master-ts-client-1
grpc-web-master-ts-client-1    | node:internal/modules/cjs/loader:1085
grpc-web-master-ts-client-1    |   throw err;
grpc-web-master-ts-client-1    |   ^
grpc-web-master-ts-client-1    | 
grpc-web-master-ts-client-1    | Error: Cannot find module '/var/www/html/python'
grpc-web-master-ts-client-1    |     at Module._resolveFilename (node:internal/modules/cjs/loader:1082:15)        
grpc-web-master-ts-client-1    |     at Module._load (node:internal/modules/cjs/loader:928:27)
grpc-web-master-ts-client-1    |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
grpc-web-master-ts-client-1    |     at node:internal/main/run_main_module:23:47 {
grpc-web-master-ts-client-1
```

This is without any changes to the code and on `master` branch, which is the only one available.

--- a/net/grpc/gateway/docker/interop_client/Dockerfile
+++ b/net/grpc/gateway/docker/interop_client/Dockerfile
@@ -29,8 +29,6 @@ RUN npm install && \
   cp index.html /var/www/html && \
   cp dist/main.js /var/www/html/dist
 
-RUN apt-get -qq update && apt-get -qq install -y python
-
 WORKDIR /var/www/html
 
 EXPOSE 8081

--- a/net/grpc/gateway/docker/prereqs/Dockerfile
+++ b/net/grpc/gateway/docker/prereqs/Dockerfile
@@ -54,6 +54,8 @@ FROM node:20.0.0-bullseye AS copy-and-build
 ARG MAKEFLAGS=-j8
 ARG BAZEL_VERSION=4.1.0
 
+RUN apt-get -qq update && apt-get -qq install -y python
+
 RUN mkdir -p /var/www/html/dist
 RUN echo "\nloglevel=error\n" >> $HOME/.npmrc
 

